spec/factories.rb

```diff
@@ -323,16 +323,7 @@
     end
   end
 
-  factory :ratifying_provider_permissions,
-          class: 'RatifyingProviderPermissions' do
-    type { 'RatifyingProviderPermissions' }
-    ratifying_provider { create(:provider) }
-    training_provider { create(:provider) }
-  end
-
-  factory :training_provider_permissions,
-          class: 'TrainingProviderPermissions' do
-    type { 'TrainingProviderPermissions' }
+  factory :provider_relationship_permissions do
     ratifying_provider { create(:provider) }
     training_provider { create(:provider) }
     setup_at { Time.zone.now }
```

Comment:
Curious about this, is it meant to distinguish permissions that are saved in the DB after import from Find but not fully populated by the providers?

Reply:
Yes - we use this to decide whether to redirect providers to set up orgs when they log in, if the orgs aren't set up yet